flink-end-to-end-tests/test-scripts/test_sql_client.sh

```diff
@@ -20,8 +20,8 @@
 set -Eeuo pipefail
 
 KAFKA_VERSION="2.2.2"
-CONFLUENT_VERSION="5.0.0"
-CONFLUENT_MAJOR_VERSION="5.0"
+CONFLUENT_VERSION="5.2.6"
```

Comment:
let's add a comment here to check the kafka<->confluent compatibility when updating the Kafka version. That didn't happen the last time.